Taskfile.yml

```diff
@@ -195,19 +200,17 @@ tasks:
       vars:
         COMPONENT: "{{.TASK}}"
 
-  log-viewer-webui:
+  log-viewer-webui-clients:
```

Comment:
The plural form here is to prepare for yscope-log-viewer UI integration, which will be in another PR by @Henry8192 .

Reply:
I'd prefer to leave it to Henry's PR.